src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -421,6 +429,69 @@ public function getXEditableChoices(FieldDescriptionInterface $fieldDescription)
         return $xEditableChoices;
     }
 
+    /**
+     * Returns a canonicalized locale for "moment" NPM library,
+     * or `null` if the locale's language is "en", which doesn't require localization.
+     *
+     * @param array $context
```

Comment:
Please remove this

Reply:
You mean the whole docblock @greg0ire?

Reply:
no, just this line, it's tautologic, whereas the rest is interesting